### PR TITLE
Grant read access for columnar metadata tables to unprivileged user

### PR DESCRIPTION
**Pull request by onurctirtir**
DESCRIPTION: Grants read access for columnar metadata tables to unprivileged user

### CODE REVIEW

File: src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql [hunk 1]
```diff
@@ -61,6 +61,9 @@ CREATE TABLE chunk (
```
Comment (onurctirtir) [inline src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql:61]: Realized that we actually allow unprivileged user to execute `alter_columnar_table_set` for a table that the user doesn't own, I think I should open a seperate pr for that ?

Comment (onurctirtir) [reply src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql:61]: https://github.com/citusdata/citus/pull/4748

**Comment by onurctirtir** [comment]
I should put that `create columnar table` test inside a pg version check block

**Comment by onurctirtir** [comment]
obviously I forgot to update `multi_extension.out` files

File: src/backend/distributed/sql/citus--10.0-1--10.0-2.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-- citus--10.0-1--10.0-2
+
+-- bump version to 10.0-2
```
Comment (marcocitus) [inline src/backend/distributed/sql/citus--10.0-1--10.0-2.sql:3]: I think this comment is mostly meant for empty upgrade files

**Review by marcocitus — APPROVED** [review]

**Pull request merged**